autoload/codefmt/jsonnetfmt.vim

```diff
@@ -0,0 +1,46 @@
+let s:plugin = maktaba#plugin#Get('codefmt')
+
+
+""
+" @private
+"
+" Formatter provider for jsonnet files using jsonnetfmt
+function! codefmt#jsonnetfmt#GetFormatter() abort
+  let l:formatter = {
+      \ 'name': 'jsonnetfmt',
+      \ 'setup_instructions': 'Install jsonnet. ' .
+          \ '(https://jsonnet.org/).'}
+
+  function l:formatter.IsAvailable() abort
+    return executable(s:plugin.Flag('jsonnetfmt_executable'))
+  endfunction
+
+  let s:supported_filetypes = ['json','jsonnet']
```

Comment:
Did you mean `l:supported_filetypes` instead of `s:`? Kind of weird to define a script-global variable that's only needed locally right below the definition.

Reply:
To be honest, I just copied this bit from https://github.com/google/vim-codefmt/blob/master/autoload/codefmt/prettier.vim :) I'll change it.